packages/common/src/Constants.ts

```diff
@@ -0,0 +1 @@
+export const NODE_CLIENT_URL = 'https://sdk-backend.prod.biconomy.io/v1'
```

Comment:
can we add all constants here specific to paymasters, account etc

Reply:
all the constants that will get used in more then one package will be moved here. Rest will stay in their own package

Reply:
ok is it done?

Reply:
for now i don't have any other constant that needed in multiple so its just NODE_CLIENT_URL for now